ui/pages/confirmations/hooks/alerts/transactions/useInsufficientBalanceAlerts.ts

```diff
@@ -31,15 +44,20 @@ export function useInsufficientBalanceAlerts(): Alert[] {
     selectTransactionValue(state, transactionId),
   );
 
+  const totalValue = sumHexes(value, ...batchTransactionValues);
+
   const { hexMaximumTransactionFee } = useSelector((state) =>
     selectTransactionFeeById(state, transactionId),
   );
 
   const nativeCurrency = useSelector(getMultichainNativeCurrency);
 
   const insufficientBalance = !isBalanceSufficient({
-    amount: value,
-    gasTotal: hexMaximumTransactionFee,
+    amount: totalValue,
+    gasTotal: multiplyHexes(
+      hexMaximumTransactionFee as Hex,
+      decimalToHex(batchTransactionCount + 1) as Hex,
```

Comment:
As these are EIP-7702 batches, there is only one transaction so one gas fee, so no need to multiply.

Reply:
oh I see, let me fix this

Reply:
PR is updated